docs/index.html

```diff
@@ -2249,7 +2249,7 @@
           -231.5279,231.248 -231.873,231.248 -0.3451,0 -104.688,
           -104.0616 -231.873,-231.248 z
         " fill="currentColor"></path></g></svg></div></div><div class="sc-kizEQm iWVhJm api-content"><div class="sc-eCApnc jlMQbh"><div class="sc-iCoGMd gLxhOh"><div class="sc-hKFxyN juinod api-info"><h1 class="sc-fujyAs sc-GvhzO cTueGk hQeXON">platform-sdk<!-- --> <span>(<!-- -->6<!-- -->)</span></h1><p>Download OpenAPI specification<!-- -->:<a download="swagger.json" target="_blank" class="sc-gsWcmt fgrWXj">Download</a></p><div class="sc-iBzEeX sc-cOifOu dFWqin OsmAW"><div class="sc-eGJWMs hgLhIW"><div class="sc-csTbgd eOJoil"><span class="sc-bXexck gRGtTW">API support<!-- -->:<!-- --> <a href="mailto:info@withthegrid.com">info@withthegrid.com</a></span> <span class="sc-bXexck gRGtTW">URL: <a href="https://withthegrid.com/">https://withthegrid.com/</a></span> <!-- --> </div></div></div><div class="sc-iBzEeX sc-cOifOu dFWqin OsmAW" data-role="redoc-summary"></div><div class="sc-iBzEeX sc-cOifOu dFWqin OsmAW" data-role="redoc-description"></div></div></div></div><div id="section/Getting-started" data-section-id="section/Getting-started" class="sc-eCApnc jlMQbh"><div class="sc-iCoGMd gLxhOh"><div class="sc-hKFxyN juinod"><h1 class="sc-fujyAs cTueGk"><a class="sc-crzoAE iUxAWq" href="#section/Getting-started" aria-label="section/Getting-started"></a>Getting started</h1></div></div></div><div id="section/Getting-started/Introduction" data-section-id="section/Getting-started/Introduction" class="sc-eCApnc jlMQbh"><div class="sc-iCoGMd gLxhOh"><div class="sc-hKFxyN juinod"><h2 class="sc-pNWdM euRMgx"><a class="sc-crzoAE iUxAWq" href="#section/Getting-started/Introduction" aria-label="section/Getting-started/Introduction"></a>Introduction</h2></div></div><div class="sc-hKFxyN bJcDWV"><div class="sc-iBzEeX sc-cOifOu dFWqin OsmAW redoc-markdown "><p>This document provides information on how to access the withthegrid application programmatically.</p>
-<p>Programmatic access to the application requires a machine account. Almost everything a user can do in the user interface of the application, can also be done with a machine account. To obtain one, contact us at <a href="mailto:&#105;&#x6e;&#x66;&#x6f;&#64;&#x77;&#x69;&#x74;&#x68;&#x74;&#x68;&#101;&#103;&#114;&#x69;&#x64;&#x2e;&#99;&#x6f;&#x6d;">&#105;&#x6e;&#x66;&#x6f;&#64;&#x77;&#x69;&#x74;&#x68;&#x74;&#x68;&#101;&#103;&#114;&#x69;&#x64;&#x2e;&#99;&#x6f;&#x6d;</a>.
```

Comment:
It keeps changing this part from unicode hex to unicode decimal. Do you see why?

Reply:
Well that is pretty strange, I have no idea why it would do either conversion. This happens when you run the build?

Reply:
yes it recognises the fact that it is an email adres and turns it into a mailto but during that process it esceapse everything and does so alternatingly between hex and decimal notations